raft/handler.go

```diff
@@ -99,10 +99,10 @@ var errNoLeaderElected = errors.New("no leader is currently elected")
 // Public interface
 //
 
-func NewProtocolManager(raftId uint16, raftPort uint16, blockchain *core.BlockChain, mux *event.TypeMux, bootstrapNodes []*enode.Node, joinExisting bool, datadir string, minter *minter, downloader *downloader.Downloader, useDns bool) (*ProtocolManager, error) {
-	waldir := fmt.Sprintf("%s/raft-wal", datadir)
-	snapdir := fmt.Sprintf("%s/raft-snap", datadir)
-	quorumRaftDbLoc := fmt.Sprintf("%s/quorum-raft-state", datadir)
+func NewProtocolManager(raftId uint16, raftPort uint16, blockchain *core.BlockChain, mux *event.TypeMux, bootstrapNodes []*enode.Node, joinExisting bool, datadir string, raftLogDir string, minter *minter, downloader *downloader.Downloader, useDns bool) (*ProtocolManager, error) {
```

Comment:
Since `datadir` is no longer used, it should be removed.

Reply:
Removed.